config/environment.js

```diff
@@ -107,7 +107,7 @@ module.exports = function(environment) {
     ENV.APP.API_HOST_URL = "http://localhost:3000";
     ENV.APP.SOCKETIO_WEBSERVICE_URL = "http://localhost:1337/goodcity";
     ENV.ADMIN_APP_HOST_URL = "http://localhost:4201";
-    ENV.BROWSE_APP_HOST_URL = "http://localhost:4203";
+    ENV.BROWSE_APP_HOST_URL = "http://localhost:4202";
```

Comment:
port `4202` is reserved for `browse` app. What is the reason for changing it ?

Reply:
Yeah, that points to browse, it was wrong